viz/cmd/uninstall.go

```diff
@@ -43,5 +45,60 @@ func uninstallRunE(ctx context.Context) error {
 		return err
 	}
 
-	return pkgCmd.Uninstall(ctx, k8sAPI, selector)
+	if err := pkgCmd.Uninstall(ctx, k8sAPI, selector); err != nil {
+		return err
+	}
+
+	// delete any HTTPRoute, AuthorizationPolicy, and Server resources created
+	// by the viz extension in any namespace
+	nses, err := k8sAPI.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
```

Comment:
I don't think we have to fetch a list of all namespaces if we just pass `k8s.io/apimachinery/pkg/apis/meta/v1.NamespaceAll` as the namespace in the queries below.